scripts/docker-entrypoint-node-1.sh

```diff
@@ -1,14 +1,13 @@
 #!/bin/bash
 
-# FIXME - check security associated with each CLI option
 /usr/local/bin/datahighway --validator \
   --unsafe-ws-external \
   --unsafe-rpc-external \
   --rpc-cors=all \
   --base-path /tmp/polkadot-chains/node-1 \
   --bootnodes /dns/${ENDPOINT_TESTNET}/tcp/30333/p2p/${BOOTNODE_NODE_ID_TESTNET} \
```

Comment:
suggest moving the following:

scripts/docker-entrypoint-node-1.sh
scripts/docker-entrypoint-node-2.sh
scripts/docker-entrypoint-node-3.sh
scripts/docker-entrypoint-node-4.sh
scripts/docker-entrypoint-node-5.sh

into a subfolder called 'harbour',

scripts/harbour/docker-entrypoint-node-1.sh
scripts/harbour/docker-entrypoint-node-2.sh
scripts/harbour/docker-entrypoint-node-3.sh
scripts/harbour/docker-entrypoint-node-4.sh
scripts/harbour/docker-entrypoint-node-5.sh